internal/storage/postgres/tree/trees_test.go

```diff
@@ -0,0 +1,225 @@
+package tree
+
+import (
+	"context"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage/postgres/mapper/generated"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage/postgres/testutils"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+var (
+	mappers TreeMappers
+	suite   *testutils.PostgresTestSuite
+)
+
+func TestMain(m *testing.M) {
+	code := 1
+	ctx := context.Background()
+	defer func() { os.Exit(code) }()
+	suite = testutils.SetupPostgresTestSuite(ctx)
+	mappers = NewTreeRepositoryMappers(
+		&generated.InternalTreeRepoMapperImpl{},
+		&generated.InternalImageRepoMapperImpl{},
+		&generated.InternalSensorRepoMapperImpl{},
+		&generated.InternalTreeClusterRepoMapperImpl{},
+	)
+	defer suite.Terminate(ctx)
+	code = m.Run()
+}
+
+func TestTreeRepository_Delete(t *testing.T) {
```

Comment:
- check for invalid id - 0 and -1
- also check if it is working if no images are set